entities/backup/descriptor_test.go

```diff
@@ -473,3 +474,82 @@ func TestShardDescriptorClear(t *testing.T) {
 	s.ClearTemporary()
 	assert.Equal(t, want, s)
 }
+
+func TestWildcardFunctionality(t *testing.T) {
```

Comment:
The purpose of the wildcard is to match on similar _class names_, not the underlying _file names_, which are opaque to the end user

Reply:
Updated accordingly!